src/components/settings/CategoriesSetting.tsx

```diff
@@ -78,9 +78,12 @@ const getCategoryUpdateInfo = (
     return categories.map((category) => category.name).join(', ');
 };
 
-export const GlobalUpdateSettingsCategories = () => {
+type CategoryIncludeField = keyof Pick<TCategory, 'includeInUpdate' | 'includeInDownload'>
+
+export const CategoriesSetting = (props: {includeField: CategoryIncludeField}) => {
```

Comment:
I think `CategoriesIncludeExcludeSetting`, `CategoriesInclusionSetting`or something like that would be more descriptive

Reply:
Yeah, I kinda just picked a name and figured it would come up during review. Naming is hard 😓